### PR TITLE
Fixed #118: Added profile links

### LINKED ISSUE #118: Add missing Profile links of speakers
Add more social profiles of speakers

### PR DESCRIPTION
**Pull request by samyakbvs**

### CODE REVIEW

**Comment by samyakbvs** [comment]
Added missing profile links

**Comment by samyakbvs** [comment]
https://samyakbvs.github.io/samyakjainbvs4-2018.fossasia.org/

File: index.html [hunk 1]
```diff
@@ -2085,14 +2097,17 @@ <h1>Speakers</h1>
                                             <a target="default" href="www.mozillaphilippines.org"><i class="icon fa fa-external-link"></i></a>
                                             <a target="default" href="https://twitter.com/bobreyes"><i class="icon social_twitter"></i></a>
                                             <a target="default" href="https://www.linkedin.com/in/bobreyes/"><i class="icon social_linkedin"></i></a>
+                                            <a target="default" href="https://www.facebook.com/robertbobreyes/
+
+                                            "><i class="icon fa fa-facebook"></i></a>
```
Comment (harshit98) [inline index.html]: Some part of above anchor tag is coming here. Kindly fix it. `">` the anchor tag should close on line 2100. Try to write this line as it is same in line no 2098, 2099.

File: index.html [hunk 2]
```diff
@@ -2207,6 +2222,8 @@ <h1>Speakers</h1>
                                         <div class="social-links vertical-align">
                                             <a target="default" href="http://libedinskylab.com/"><i class="icon fa fa-external-link"></i></a>
                                             <a target="default" href="https://www.linkedin.com/in/camilo-libedinsky-4a4a4126/"><i class="icon social_linkedin"></i></a>
+                                            <a target="default" href="https://twitter.com/libedinskylab?lang=en"><i class="icon social_twitter"></i></a>
+
```
Comment (harshit98) [inline index.html]: Remove extra line.

File: index.html [hunk 3]
```diff
@@ -2266,7 +2284,7 @@ <h1>Speakers</h1>
                         <div class="col-md-3 col-sm-6 speaker-column">
                             <div class="speaker">
                                 <div class="image-holder">
-                                    <img class="background-image" alt="Matthew Treinish" src="img/MatthewTreinish.jpg">
+                                    <img class="background-image" alt="Nan Xiao" src="img/MatthewTreinish.jpg">
```
Comment (harshit98) [inline index.html]: Why you've changed `alt` name? Keep it same as it was earlier.

File: index.html [hunk 4]
```diff
@@ -2281,37 +2299,22 @@ <h1>Speakers</h1>
                         <div class="col-md-3 col-sm-6 speaker-column">
                             <div class="speaker">
                                 <div class="image-holder">
-                                    <img class="background-image" alt="Daniel Gultsch" src="img/danielgultsch.jpg">
+                                    <img class="background-image" alt="Nan Xiao" src="img/danielgultsch.jpg">
```
Comment (harshit98) [inline index.html]: Don't change `alt` name here. Keep it as it was earlier.

File: index.html [hunk 5]
```diff
@@ -2281,37 +2299,22 @@ <h1>Speakers</h1>
...
                                     <div class="hover-state text-center preserve3d">
                                         <div class="social-links vertical-align">
                                             <a target="default" href="https://gultsch.de/"><i class="icon fa fa-external-link"></i></a>
                                             <a target="default" href="https://twitter.com/iNPUTmice"><i class="icon social_twitter"></i></a>
                                             <a target="default" href="https://github.com/inputmice/"><i class="icon fa fa-github"></i></a>
+                                            <a target="default" href="https://twitter.com/road2stat?lang=en
```
Comment (harshit98) [inline index.html]: Try to write it in one line.

File: index.html [hunk 6]
```diff
@@ -2281,37 +2299,22 @@ <h1>Speakers</h1>
...
+                                            "><i class="icon social_twitter"></i></a>
+
```
Comment (harshit98) [inline index.html]: Remove extra 1 line.

File: index.html [hunk 7]
```diff
@@ -3082,9 +3085,9 @@ <h1>Community Partner</h1>
                             <ul class="contact-methods">
                                 <li><i class="icon pe-7s-mail"></i><span><a id="mailAdd" href="mailto:fossasia@googlegroups.com" target="_self" style="color:#777777">fossasia@googlegroups.com</a></span></li>
                                 <li><i class="icon pe-7s-phone"></i><span>+65 83435672</span></li>
-                                <li><i class="icon pe-7s-map-marker"></i><span>FOSSASIA</span></li>
+                                <li><i style="color: #000000;"class="icon pe-7s-map-marker"></i><span>FOSSASIA</span></li>
```
Comment (harshit98) [inline index.html]: Remove these changes.

File: index.html [hunk 8]
```diff
@@ -3082,9 +3085,9 @@ <h1>Community Partner</h1>
...
                                 <li><span>Ms. DANG Hong Phuc</span></li>
-                                <li><span>6 Eu Tong Sen Street #05-07 The Central, Singapore 059817</span></li>
+                                <li><span><a id="mailAdd" href="https://www.google.co.in/maps/search/6+Eu+Tong+Sen+Street+%2305-07+The+Central,+Singapore+059817/@1.2889009,103.8444613,17z/data=!3m1!4b1?hl=en" target="_self" style="color:#777777">6 Eu Tong Sen Street #05-07 The Central, Singapore 059817</a></span></li>
```
Comment (harshit98) [inline index.html]: Remove these changes.

**Comment by samyakbvs** [comment]
Closes #118

**Comment by harshit98** [comment]
@samyakbvs Squash your commits.

**Comment by samyakbvs** [comment]
Squashed

**Comment by samyakbvs** [comment]
@harshit98

File: index.html [hunk 9]
```diff
@@ -3082,7 +3080,7 @@ <h1>Community Partner</h1>
                             <ul class="contact-methods">
                                 <li><i class="icon pe-7s-mail"></i><span><a id="mailAdd" href="mailto:fossasia@googlegroups.com" target="_self" style="color:#777777">fossasia@googlegroups.com</a></span></li>
                                 <li><i class="icon pe-7s-phone"></i><span>+65 83435672</span></li>
-                                <li><i class="icon pe-7s-map-marker"></i><span>FOSSASIA</span></li>
+                                <li><i>"class="icon pe-7s-map-marker"></i><span>FOSSASIA</span></li>
```
Comment (0x48piraj) [inline index.html]: remove the `"`

**Comment by samyakbvs** [comment]
Commits have been squashed as well

**Comment by samyakbvs** [comment]
@harshit98  changes made pls review

**Comment by STEdirisuriya** [comment]
Please give a description

**Comment by samyakbvs** [comment]
I did scroll above

File: index.html [hunk 10]
```diff
@@ -1634,23 +1634,26 @@ <h2>Bunnie Huang</h2>
                             <div class="speaker-with-bio">
                                 <div class="speaker">
                                     <div class="image-holder">
-                                        <img class="background-image" alt="Carsten Haitzler" src="img/CarstenHaitzler.jpg">
+                                        <img class="background-image" alt="Masakazu Takasu" src="img/MasakazuTakasu.jpg">
```
Comment (sayak119) [inline index.html]: Please refer https://github.com/fossasia/2018.fossasia.org/issues/111

File: index.html [hunk 11]
```diff
@@ -2287,31 +2302,14 @@ <h1>Speakers</h1>
...
                                             <a target="default" href="https://github.com/inputmice/"><i class="icon fa fa-github"></i></a>
+                                            <a target="default" href="https://twitter.com/road2stat?lang=en"><i class="icon social_twitter"></i></a>                                          
                                         </div>
                                     </div>
                                 </div>
                                 <span class="speaker-name">Daniel Gultsch</span>
                                 <span>Conversations</span>
                             </div>
                         </div>
-                        <div class="col-md-3 col-sm-6 speaker-column">
```
Comment (sayak119) [inline index.html]: Refer #111

Comment (samyakbvs) [reply index.html]: what are you talkking about?

Comment (0x48piraj) [reply index.html]: He's saying to look into that specific issue @samyakbvs 
You have conflicted that issue.

Comment (sayak119) [reply index.html]: @samyakbvs  Don't delete anything with respect to Carsten Haitzler part and Masakazu Takasu part. You can add more profile links if you want.

**Comment by samyakbvs** [comment]
@sayak119

**Comment by samyakbvs** [comment]
@vaibhavsingh97

**Comment by samyakbvs** [comment]
@harshit98, @vaibhavsingh97 can you pls do something about merging this as this has been pending for a long time and so is my task because of an unmerged PR. So pls you guys merge it or if you can pls ask @mariobehling to view this
Thanks!

File: index.html [hunk 12]
```diff
@@ -206,11 +206,11 @@ <h1>Check out the schedule with 200+ sessions from speakers around the world fro
 
 
 			<a id="schedule" class="in-page-link"></a>
-
+
```
Comment (abishekvashok) [inline index.html]: Unwanted spaces

File: index.html [hunk 13]
```diff
@@ -206,11 +206,11 @@ <h1>Check out the schedule with 200+ sessions from speakers around the world fro
 
 
 			<a id="schedule" class="in-page-link"></a>
-
+			
 			<section class="schedule">
 				<div class="container">
 					<div class="row">
-
+
```
Comment (abishekvashok) [inline index.html]: Same here

File: index.html [hunk 14]
```diff
@@ -219,38 +219,38 @@ <h1>Exhibition Program<br/>
 								<li>
 									<div class="schedule-title">
 										<span class="time">March 22, 12:00 AM - 6:00 PM</span>
-										<span class="title">Exhibition, Career Fair, VIP Tour</span>
+										<span class="title">Exhibition, Career Fair, VIP Tour</span>										
 									</div>
-
+
```
Comment (abishekvashok) [inline index.html]: Again some unwanted spaces

File: index.html [hunk 15]
```diff
@@ -219,38 +219,38 @@ <h1>Exhibition Program<br/>
...
+									
 									<div class="schedule-text">
 										<p>12:00 PM Exhibition & Career Fair Doors Open<br/>
 										1:00 PM Welcome Snacks<br/>
 										3:00 PM VIP Tour at Exhibition<br/>
 										3:00 PM Career Booth Crawl<br/>
 										6:00 PM Exhibition Doors Close</p>
 									</div>
-
+
```
Comment (abishekvashok) [inline index.html]: trailing white spaces

File: index.html [hunk 16]
```diff
@@ -219,38 +219,38 @@ <h1>Exhibition Program<br/>
...
-
+									
 									<div class="marker-pin">
 										<div class="top"></div>
 										<div class="middle"></div>
 										<div class="bottom"></div>
 									</div>
 								</li>
-
+
```
Comment (abishekvashok) [inline index.html]: here as well

File: index.html [hunk 17]
```diff
@@ -219,38 +219,38 @@ <h1>Exhibition Program<br/>
...
-
+								
 								<li>
 									<div class="schedule-title">
 										<span class="time">March 23, 10:00 AM - 6:00 PM</span>
-										<span class="title">Exhibition, Business Lunch and Afternoon Booth Tour</span>
+										<span class="title">Exhibition, Business Lunch and Afternoon Booth Tour</span>										
 									</div>
-
+
```
Comment (abishekvashok) [inline index.html]: same here

File: index.html [hunk 18]
```diff
@@ -219,38 +219,38 @@ <h1>Exhibition Program<br/>
...
+									
 									<div class="schedule-text">
 										<p>9:30 AM Exhibitors Breakfast Snacks<br/>
 										10:00 AM Exhibition Doors Open<br/>
 										12:00 Business Lunch (Invitation)<br/>
 										5:00 PM Happy Hour Booth<br/>
 										6:00 Exhibition Doors Close</p>
 									</div>
-
+
```
Comment (abishekvashok) [inline index.html]: same here

File: index.html [hunk 19]
```diff
@@ -3064,7 +3078,7 @@ <h1>Community Partner</h1>
                     <div class="row ui-sortable">
                         <div class="col-sm-2">
                             <ul class="menu">
-                                <li><a target="_self" href="https://blog.fossasia.org">Blog</a></li>
+                                <li><a target="_self" href="https://blog.fossasia.org">Blog</a></li>
```
Comment (abishekvashok) [inline index.html]: trailing whitespace

**Comment by 0x48piraj** [comment]
@mariobehling Please revert this PR. It has many conflicts !

File: index.html [hunk 20]
```diff
@@ -1993,6 +1998,11 @@ <h1>Speakers</h1>
                                     <div class="hover-state text-center preserve3d">
                                         <div class="social-links vertical-align">
                                             <a target="default" href="https://github.com/arynchoong"><i class="icon fa fa-github"></i></a>
+                                            <a target="default" href="https://www.facebook.com/arynchoong
+                                            "><i class="icon fa fa-facebook"></i></a>
```
Comment (0x48piraj) [inline index.html]: @mariobehling See this !

**Comment by 0x48piraj** [comment]
Please merge this PR first, It's clean https://github.com/fossasia/2018.fossasia.org/pull/135

**Comment by duttaditya18** [comment]
This does have a lot of problems.

**Comment by 0x48piraj** [comment]
@duttaditya18 If @mariobehling will not revert it, I'll solve this.
Please don't send a PR. :wink:

**Comment by mariobehling** [comment]
Lots of issues when deployed.

**Comment by 0x48piraj** [comment]
I think, we should Revert the changes @mariobehling

**Comment by 0x48piraj** [comment]
Please see my Pull Request, It's bug free and also is approved :+1:  #135

**Pull request merged**